### DISCUSSION TITLE
Remove microphone feedback

### DISCUSSION THREAD

**Discussion by TraceLTRC** [wwmm/easyeffects#1442] [Q&A]
I have easyeffects installed on my laptop. Because the mic and speakers are close, my microphone picks up the sound from my speakers. I want to know what effect I can use to remove the sounds from my speakers that is picked up by my microphone. Thank you!

**Comment by wwmm** [comment]
The plugin for this kind of situation is the `Echo Canceller`. There is one in EasyEffects based on the `SpeexDSP` library. But have in mind that this plugin is far from perfect. Depending on the situation it works. But sometimes it doesn't. I have to find a library that is easier to use.

**Reply by wwmm** [reply]
PipeWire has a built-in echo canceller that uses a different library. If the one on EasyEffects does not work this is something to consider.
